src/content/en/tools/chrome-devtools/beginners/css.md

```diff
@@ -2,82 +2,114 @@ project_path: /web/tools/_project.yaml
 
 ## Goals {: #goals }
 
 By the end of this tutorial, you will understand:
 
-* How CSS can be used to style webpages.
-* How Chrome DevTools can help you when you're working CSS.
-* The difference between CSS and CSS Frameworks.
+* How to use CSS to style a web page.
+* How to use Chrome DevTools to experiment with CSS.
+* The difference between CSS and CSS frameworks.
 
 You'll also have a real website!
 
 ## Prerequisites {: #prerequisites }
 
 Before attempting this tutorial, complete the following prerequisites:
 
-* Complete the previous tutorial on HTML and DOM or complete *Setup* below.
-* Download the [Google Chrome][chrome]{: .external } web browser. This tutorial uses a set of web development tools, 
-called Chrome DevTools, that are built into Google Chrome. 
+* Comlete [Get Started with HTML and the DOM][HTML] or make sure that you have an understanding
+  of HTML and the DOM similar to what's taught in that tutorial.
+* Download the [Google Chrome][chrome]{: .external } web browser. This tutorial uses a set of
+  web development tools, called Chrome DevTools, that are built into Google Chrome. 
+
 [chrome]: https://www.google.com/chrome/
 
 ## Set up your code {: #setup }
 
- 
-
 In order to start creating your site, you need to set up your code:
 
 1. **If you have already completed the first tutorial in this series, skip this section! 
    Continue using your code from the last tutorial, 
    [Getting Started with HTML and the DOM](/web/tools/chrome-devtools/beginners/html).**
 2. Open the [source code](https://glitch.com/edit/#!/dfb2). A code editor called 
-   Glitch shows a page called `index.html`. This will be called the editing tab.
+   Glitch shows a page called `index.html`. This tab of your browser will be called the
+   **editing tab**.
 
-`(TODO: A screenshot showing the action.)`
+     <figure>
+       <img src="imgs/css/setup1.png"
+            alt="The source code."/>
+       <figcaption>
+         <b>Figure X</b>. The source code
+       </figcaption>
+     </figure>
 
 2. Click **dfb2**. A menu pops up.
 
-`(TODO: A screenshot showing the menu)`
+     <figure>
+       <img src="imgs/css/setup2.png"
+            alt="The editor tab."/>
+       <figcaption>
+         <b>Figure X</b>. The editor tab
+       </figcaption>
+     </figure>
 
 3. Click **Remix This**. Glitch creates a copy of the project that you can 
-   edit. Note that the name of the new project will be randomly generated
-   and not dfb2. The content is the same, but the name on the top-left has changed.
-4. Click **Show Live**. Another tab opens with view of what your site 
-   currently looks like. This will be called the **live tab**.
-
-`(TODO: A screenshot showing the result.)`
+   edit. Note that the name of the new project will be randomly generated.
+4. Click **Show Live**. Another tab opens with a live view of your site.
+   This tab of your browser will be called the **live tab**.
 
+     <figure>
+       <img src="imgs/css/setup3.png"
```

Comment:
Notice the filename format. Use the anchor of the section, followed by an incrementally-increasing number. E.g. the anchor for this section is `#setup` so the filename is `setup3.png`.

Also, make sure to run all images through tinypng.com.